canvas2D/src/Engine/babylon.text2d.ts

```diff
@@ -404,6 +418,18 @@
             this._textSize           = null;
             this.text                = text;
             this.size                = (settings.size==null) ? null : settings.size;
+            
+            if(settings.textAlignH != null){
+                this.textAlignH = settings.textAlignH;
+            }
+
+            if(settings.textAlignV != null){

```

Comment:
textAlignV/H and this._wordWrap shoudl always be initialized in the object to keep the object's layout consistent with other instances.